e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -799,6 +788,39 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
       horizontal: -100,
     });
   });
+
+  it("should not crash notebook when metric is used as an aggregation and breakout is applied (metabase#40553)", () => {
+    createMetric({
+      name: "Revenue",
+      description: "Sum of orders subtotal",
+      table_id: ORDERS_ID,
+      definition: {
+        "source-table": ORDERS_ID,
+        aggregation: [["sum", ["field", ORDERS.SUBTOTAL, null]]],
+      },
+    }).then(({ body }) => {
+      cy.wrap(body.id).as("metricId");
+    });
+
+    cy.get("@metricId").then(metricId => {
```

Comment:
No need to wrap this if you're going to use it straight away. Feel free to write the rest of the test in the `.then` block.

Reply:
it's a cleaner approach imo to separate metric creation and metricId usage

Reply:
I would agree if there were more places/instances of this code that need to access this value later on in the test. But this is just creating more work for Cypress and is more confusing to read.

If you really wanted to have a clear separation between the _arrange_, _act_, and _assert_ phases, then you should've grouped together create metric and create question as the _arrange_.

Reply:
I agree, I actually missed that not whole code goes to `then`, so you're right here, fixed